docs/source/examples.rst

```diff
@@ -84,6 +84,14 @@ those `vision datasets here <https://github.com/pytorch/vision/tree/main/torchvi
 Note that these implementations are currently in the prototype phase, but they should be fully supported
 in the coming months. Nonetheless, they demonstrate the different ways DataPipes can be used for data loading.
 
+Caltech 256 and Microsoft COCO (community example)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Listing and loading data from AIS buckets (buckets that are not 3rd party backend-based) and remote cloud buckets (3rd party 
+backend-based cloud buckets) using `AISFileLister <https://pytorch.org/data/main/generated/torchdata.datapipes.iter.AISFileLister.html#aisfilelister>`_ and `AISFileLoader <https://pytorch.org/data/main/generated/torchdata.datapipes.iter.AISFileLoader.html#torchdata.datapipes.iter.AISFileLoader>`_.
+
+Here is an `example which uses AISIO DataPipe <https://github.com/pytorch/data/blob/main/examples/vision/aisio_usage_example.ipynb>`_ for the `Caltech-256 Object Category Dataset <https://data.caltech.edu/records/20087>`_ containing 256 object categories and a total 
```

Comment:
Let's create another subfolder for aistore specific examples rather than using `vision` ones.

Reply:
done